### ISSUE TITLE
Screen Reader Issue with Object Autocomplete Dropdown Field for Data Import

### ISSUE THREAD

**Issue by rlledder** [tprouvot/Salesforce-Inspector-reloaded#628]
**Make sure to read the troubleshooting section before creating an issue**

- [X] I've read the [common issues](https://tprouvot.github.io/Salesforce-Inspector-reloaded/troubleshooting/) and the one I'm about to create is not one of those documented.
- [X] I've read the [release note](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/releaseCandidate/CHANGES.md) and the one I'm about to create is not already fixed.

**Describe the bug**
When using Salesforce Inspector Reloaded with a screen reader like NVDA in Chrome, the autocomplete dropdown for the Object field when doing data import fails to read autocomplete suggestions properly. Instead, it continues to repeat the text input. The issue appeared when Chrome updated, and there doesn't appear to be a problem in Firefox.

**To Reproduce**
1. Log into Salesforce with NVDA active
2. Press Alt+I to open the inspector
3. Select the Data Import button
4. Tab into the Object field
5. Begin typing the name of the object you want to bring up
6. Use up and down arrows to move through the autocomplete options
7. NVDA repeatedly reads the text input you typed instead of the actual autocomplete options, making it impossible to correctly select your object

**Expected behavior**
NVDA should read the options that are coming up in autocomplete when the up and down arrows are used to navigate through them. This works in Firefox, and previously worked in Chrome as well, but it broke when Chrome updated recently.

**Screenshots**
N/A, since this is a screen reader problem.

**Desktop (please complete the following information):**
 - OS: Windows 10 and 11
 - Browser Chrome (not Firefox, not tested in Edge)
 - Version Multiple

**Additional context**
As mentioned, this seems to have broken when Chrome updated.
